### DISCUSSION TITLE
Add option to customize the snippet used in Slack notifications when a document is updated.

### DISCUSSION THREAD

**Discussion by theodejager** [outline/outline#5754] [Ideas]
Would it be possible to customize the content of the document updated slack notification? Or perhaps make it do a small summary of the changes?

**Comment by tommoor** [comment]
Yes, there is an outstanding issue for this (https://github.com/outline/outline/issues/5295) – I started looking at it, but reliably representing a diff in Slack's limited markdown language isn't easy.
